src/libs/actions/Report.js

```diff
@@ -435,6 +435,7 @@ function reconnect(reportID) {
                 value: {
                     isLoadingReportActions: true,
                     isLoadingMoreReportActions: false,
+                    reportName: lodashGet(allReports, [reportID, 'reportName'], CONST.REPORT.DEFAULT_REPORT_NAME),
```

Comment:
I commented below, but this is not the part that solves the issue. I added this because unit tests were failing and when investigating this was what fixed the unit test issue and brings `reconnectToReport` functionality in-line with `openReport`.